pages/sponsor-us-page.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import Link from "next/link";
+import "../app/globals.css";
+import Footer from "../components/footer";
+import NavBar from "@/components/nav-bar/nav-bar";
+
+function SponsorUsPage() {
+	return (
+		// main container
+		<div className="flex flex-col items-center justify-center min-h-screen text-center w-5/6 mx-auto pt-36">
+			<NavBar showOnScroll={false} />
+			<div className="flex flex-col items-center mb-8">
+				<h3 className="text-5xl font-bold mb-4">SPONSOR US!</h3>
+				<Link
+					href="https://tinyurl.com/hackrpi-donation"
+					className="btn btn-primary rounded-full bg-hackrpi-primary-blue border-none hover:bg-hackrpi-primary-blue-dark px-10 custom-btn-height"
+				>
+					Donate
+				</Link>
+			</div>
+			<div className="flex justify-center w-full">
+				<div className="flex flex-col items-center w-full lg:w-1/2 p-2" style={{ marginRight: "20px" }}>
+					<h1 className="text-2xl font-bold mb-4">Want to support?</h1>
+					<iframe
+						className="w-full h-[60vh] lg:h-[60vh]" // Adjusted height
```

Comment:
This comment is a bit redundant, also its redundant to have a media query that scales to the same size as its orignal sizes. Lets also use css aspect ratio to set the height so that the height scales with the width.
```suggestion
						className="w-full aspect-[8.5/11]"
```

Reply:
Unsure if that code broke the height or its because of invalid URL.